Altis_Life.Altis/core/actions/fn_pickupItem.sqf

```diff
@@ -27,7 +27,7 @@ if (playerSide isEqualTo west && _illegal isEqualTo 1) exitWith {
 };
 
 life_action_delay = time;
-_diff = [(_itemInfo select 0),(_itemInfo select 1),life_carryWeight,life_maxWeight] call life_fnc_calWeightDiff;
+private _diff = [(_itemInfo select 0),(_itemInfo select 1),life_carryWeight,life_maxWeight] call life_fnc_calWeightDiff;
```

Comment:
This can be connected with above params (One more reason to use params). Do _itemInfo element 1 in params statement.